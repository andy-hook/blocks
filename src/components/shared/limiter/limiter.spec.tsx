import React from "react"
import renderer from "react-test-renderer"

import Limiter from "./limiter"

describe("<Limiter />", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Limiter>Child text</Limiter>).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it("renders as small", () => {
    const tree = renderer
      .create(<Limiter size="small">Child text</Limiter>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })

  it("renders as medium", () => {
    const tree = renderer
      .create(<Limiter size="medium">Child text</Limiter>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })

  it("renders as large", () => {
    const tree = renderer
      .create(<Limiter size="large">Child text</Limiter>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
