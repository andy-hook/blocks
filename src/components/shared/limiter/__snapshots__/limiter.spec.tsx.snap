// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Limiter /> renders as large 1`] = `
<div
  className="sc-htpNat jgnjPb"
>
  Child text
</div>
`;

exports[`<Limiter /> renders as medium 1`] = `
<div
  className="sc-bwzfXH jXoYnq"
>
  Child text
</div>
`;

exports[`<Limiter /> renders as small 1`] = `
<div
  className="sc-bdVaJa eriKRN"
>
  Child text
</div>
`;

exports[`<Limiter /> renders correctly 1`] = `
<div
  className="sc-bwzfXH jXoYnq"
>
  Child text
</div>
`;
